handler/src/main/java/io/netty/handler/ssl/SslContext.java

```diff
@@ -1157,7 +1159,7 @@ private static PrivateKey getPrivateKeyFromByteBuffer(ByteBuf encodedKeyBuf, Str
     protected static TrustManagerFactory buildTrustManagerFactory(
             File certChainFile, TrustManagerFactory trustManagerFactory)
             throws NoSuchAlgorithmException, CertificateException, KeyStoreException, IOException {
-        return buildTrustManagerFactory(certChainFile, trustManagerFactory, KeyStore.getDefaultType());
```

Comment:
Why this change?

Reply:
because we call `KeyStore.getDefaultType()` in the method itself if `null` is passed